packages/gatsby-source-wordpress/src/steps/ingest-remote-schema/is-excluded.js

```diff
@@ -27,15 +28,16 @@ const fieldIsExcludedOnParentType = ({ pluginOptions, field, parentType }) => {
     field => field.name === `nodes`
   )
 
-  const parentTypeNodesFieldTypeName = findTypeName(parentTypeNodesField?.type)
+  const parentTypeNameSettings = getTypeSettingsByType(parentType)
+  const parentTypeNodesFieldTypeNameSettings = getTypeSettingsByType(
+    parentTypeNodesField?.type
+  )
```

Comment:
This is the proper way to look up plugin options type settings, not sure why I did it the other way before ;p